docs/topics/dokka-migration.md

```diff
@@ -148,6 +147,8 @@ options according to your project setup:
   documentedVisibilities.set(
       setOf(VisibilityModifier.Public)
   )
+  // or
+  documentedVisibilities(VisibilityModifier.Public)
```

Comment:
DGPv2 has a utility function for adding documented visibilities

https://github.com/Kotlin/dokka/blob/220922378e6c68eb148fda2ec80528a1b81478c9/dokka-runners/dokka-gradle-plugin/src/main/kotlin/engine/parameters/HasConfigurableVisibilityModifiers.kt#L14-L16